downstream/assemblies/platform/assembly-migrate-isolated-execution-nodes.adoc

```diff
@@ -0,0 +1,40 @@
+ifdef::context[:parent-context: {context}]
+
+[id="assembly-migrate-isolated-execution-nodes"]
+= Upgrading isolated nodes to execution nodes
+
+[role="_abstract"]
+When upgrading from version 1.x to the latest version of the {PlatformName}, platform administrators will need to migrate the data from their legacy isolated nodes into execution nodes, which will be necessary to deploy automation mesh.
+
+To do so, this guide outlines the steps to perform a side-by-side migration. This migration involves duplicating your original {PlatformName} instance with all its data and nodes, then performing an upgrade on the duplicate instance. This ensures that the data on your original automation environment will remain untouched while performing your upgrade and migration process.
+
+Below is a summary of steps needed to complete the migration process:
```

Comment:
Suggest rewording:
The migration process involves the following steps:

Can you link from this list to the procedure for each of the steps?